GPflow/gpflowrc

```diff
@@ -15,3 +15,6 @@ ekern_quadrature = warn
 [profiling]
 dump_timeline = False
 dump_tensorboard = False
+output_file_name = timeline
```

Comment:
Is this not implied by it being under the profiling section? The property is accessed using `GPflow.settings.profiling.output_file_name`.